sdcm/fill_db_data.py

```diff
@@ -2972,20 +2996,10 @@ def fill_db_data(self):
             # Create all tables according the above list
             self.cql_create_tables(session)
 
-            # Insert data to the tables according to the "inserts" and flush to disk in several cases (nodetool flush)
-            self.cql_insert_data_to_tables(session, session.default_fetch_size)
-
     def verify_db_data(self):
         # Prepare connection
         node = self.db_cluster.nodes[0]
         with self.cql_connection_patient(node, keyspace='keyspace_fill_db_data') as session:
             # override driver consistency level
             session.default_consistency_level = ConsistencyLevel.QUORUM
             self.run_db_queries(session, session.default_fetch_size)
-
-    def clean_db_data(self):
```

Comment:
why did you remove it?

Reply:
It's never used.
This PR replaced the clean_db_data() by truncate